projects/packages/assets/src/class-script-data.php

```diff
@@ -108,22 +111,24 @@ protected static function get_admin_script_data() {
 
 		$data = array(
 			'site' => array(
-				'admin_url'    => esc_url_raw( admin_url() ),
-				'date_format'  => get_option( 'date_format' ),
-				'icon'         => self::get_site_icon(),
-				'is_multisite' => is_multisite(),
-				'plan'         => array(
+				'admin_url'         => esc_url_raw( admin_url() ),
+				'date_format'       => get_option( 'date_format' ),
+				'icon'              => self::get_site_icon(),
+				'is_multisite'      => is_multisite(),
+				'is_wpcom_platform' => ( new Host() )->is_wpcom_platform(),
+				'plan'              => array(
 					// The properties here should be updated by the consumer package/plugin.
 					// It includes properties like 'product_slug', 'features', etc.
 					'product_slug' => '',
 				),
-				'rest_nonce'   => wp_create_nonce( 'wp_rest' ),
-				'rest_root'    => esc_url_raw( rest_url() ),
-				'title'        => self::get_site_title(),
-				'wp_version'   => $wp_version,
-				'wpcom'        => array(
+				'rest_nonce'        => wp_create_nonce( 'wp_rest' ),
+				'rest_root'         => esc_url_raw( rest_url() ),
+				'suffix'            => ( new Status() )->get_site_suffix(),
+				'title'             => self::get_site_title(),
+				'wp_version'        => $wp_version,
+				'wpcom'             => array(
 					// This should contain the connected site details like blog_id, is_atomic etc.
-					'blog_id' => 0,
+					'blog_id' => Manager::get_site_id( true ),
```

Comment:
This part isn't needed for the fix but I think it is fine to assign the `blog_id` early.

Reply:
It was adding a circular dependency so didn't made this change.